build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.jpms-module-dependencies.gradle.kts

```diff
@@ -39,6 +39,10 @@ javaModuleDependencies {
     moduleNameToGA.put("com.swirlds.platform.core", "com.swirlds:swirlds-platform-core")
     moduleNameToGA.put("com.swirlds.platform.gui", "com.swirlds:swirlds-platform-gui")
     moduleNameToGA.put("com.swirlds.test.framework", "com.swirlds:swirlds-test-framework")
+    moduleNameToGA.put(
+        "com.swirlds.merkle.test.fixtures",
+        "com.swirlds:swirlds-merkle|com.swirlds:swirlds-merkle-test-fixtures"
```

Comment:
What does this pipe mean?

Reply:
That is needed to define the test-fixtures as a module. That will change in future. For more details ask @jjohannes